post-generate.rhai

```diff
@@ -0,0 +1,5 @@
+// Rename template files.
+file::rename("Cargo.toml.template", "Cargo.toml");
```

Comment:
If this happens *after* the generation, are the placeholders still replaced?

Reply:
Yes, `cargo-generate` will replace the placeholders in `Cargo.toml.template`, and then renaming the file will overwrite the template-local version `Cargo.toml`.